src/main/java/de/tum/in/www1/artemis/web/rest/StatisticsResource.java

```diff
@@ -0,0 +1,172 @@
+package de.tum.in.www1.artemis.web.rest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import de.tum.in.www1.artemis.domain.enumeration.SpanType;
+import de.tum.in.www1.artemis.repository.*;
+import de.tum.in.www1.artemis.service.*;
+
+/**
+ * REST controller for managing user statistics.
+ */
+@RestController
+@RequestMapping("/api")
+@PreAuthorize("hasRole('ADMIN')")
+public class StatisticsResource {
+
+    private final Logger log = LoggerFactory.getLogger(StatisticsResource.class);
+
+    private final StatisticsService service;
+
+    public StatisticsResource(StatisticsService service) {
+        this.service = service;
+    }
+
+    /**
+     * GET management/statistics/users : get the amount of logged in user in the last "span" days.
+     *
+     * @param span the period of which the amount should be calculated
+     * @return the ResponseEntity with status 200 (OK) and the amount of users in body, or status 404 (Not Found)
+     */
+    @GetMapping("management/statistics/users")
+    @PreAuthorize("hasAnyRole('INSTRUCTOR', 'ADMIN')")
+    public ResponseEntity<Integer> getloggedUsers(@RequestParam long span) {
+        log.debug("REST request to get user login count in the last {} days", span);
+        return ResponseEntity.ok(this.service.getLoggedInUsers(span));
+    }
+
+    /**
+     * GET management/statistics/activeUsers : get the amount of active users which made a submission in the last "span" days.
+     *
+     * @param span the period of which the amount should be calculated
+     * @return the ResponseEntity with status 200 (OK) and the amount of submissions in body, or status 404 (Not Found)
+     */
+    @GetMapping("management/statistics/activeUsers")
```

Comment:
Not really sure here but does Artemis use came case for urls? Wouldn't `active-users` be a bit better?

Reply:
Thats actually true, yes. If its ok, i would change this when i refactor this method. (Which will be soon) Thanks for the tip! The only route actively used right now is "management/statistics/submission" which should be fine